### PR TITLE
Gh pages

### PR DESCRIPTION
**Pull request by zahariasp**
New buttons created

### CODE REVIEW

**Comment by gaeun** [comment]
Can you resolve the merge conflicts?

**Comment by robbyoconnor** [comment]
You have conflicts. 

Also, you need to make your photo `240x240`.

**Comment by robbyoconnor** [comment]
Also your PR title needs to be more descriptive @zahariasp.

**Comment by robbyoconnor** [comment]
@zahariasp you need to resolve these merge conflicts.

File: index.html [hunk 1]
```diff
@@ -328,7 +328,7 @@
                     
                     <div id="issues-categories-wrapper">
                         <div class="issues-category" id="unlabeled-category">
-                            <div class="button"><a href="//github.com/fossasia/gci16.fossasia.org/issues?q=is%3Aopen+is%3Aissue+no%3Alabel">Unlabeled</a></div>
+                            <div class="button btn btn-lg" mouseover="buttons()" id="Zahari"><a href="//github.com/fossasia/gci16.fossasia.org/issues?q=is%3Aopen+is%3Aissue+no%3Alabel">Unlabeled</a></div>
                         </div>
```
Comment (abishekvashok) [inline index.html:332]: Dont name IDs after your names

File: javascripts/main.js [hunk 2]
```diff
@@ -134,3 +134,8 @@ function onScroll() {
     }
   });
 }
+
+function buttons(){
+  var btn = document.getElementById('Zahari');
+  btn.style.opacity = 0.6;
```
Comment (abishekvashok) [inline javascripts/main.js:140]: Use css for opacities see the pseudo selectors

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Comment by abishekvashok** [comment]
You can check the merge conflicts resolving tutorials in the contributing.md file or you could ping me on slack @zahariasp

**Comment by robbyoconnor** [comment]
@Abhi2424shek -- he's gonna have to resolve conflicts -- his code could change significiantly...so the review was premature.

**Review by robbyoconnor — CHANGES_REQUESTED** [review]
Fix merge conflicts, resize your photo

**Comment by robbyoconnor** [comment]
@zahariasp I am closing this PR due to lack of activity.

**Comment by robbyoconnor** [comment]
Please open a new one when you are ready.

**Pull request closed without merging**